### PR TITLE
i1853: Refactor admin forgotten password

### PR DESCRIPTION
**Pull request by MartinEden**
https://vimc.myjetbrains.com/youtrack/issue/VIMC-1853

### CODE REVIEW

File: app/src/main/admin/components/AdminForgottenPasswordPage.tsx [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
+            urlFragment: "forgotten-password/"
+        }
     }
 
     render(): JSX.Element {
-        return <Page page={this}>
-            <ForgottenPasswordForm />
-        </Page>;
+        return <div>
+            <AdminPageHeader/>
```
Comment (littledice) [inline app/src/main/admin/components/AdminForgottenPasswordPage.tsx]: Am I right in thinking that now everything is redux we can move this `AdminpageHeader` component to the router (similarly for the other portals) and then we could have a single `ForgottenPasswordPageComponent` that all 3 portals route to? I think the only difference between these 3 pages is the header...

File: app/src/main/admin/components/AdminRouter.tsx [hunk 2]
```diff
@@ -27,7 +26,7 @@ interface AdminRouterProps {
 export const AdminRouter : React.StatelessComponent<AdminRouterProps> = (props: AdminRouterProps) => {
 
     const loggedIn = <Switch>
-        <Route exact path="/" component={MainMenu}/>
+        <Route exact path="/" component={MainMenuNew}/>
```
Comment (littledice) [inline app/src/main/admin/components/AdminRouter.tsx]: We should also go through and rename all the `xNew` components to `x`, but maybe there's another PR for this?

Comment (MartinEden) [reply app/src/main/admin/components/AdminRouter.tsx]: Yup, that's the last PR

File: app/src/main/admin/components/AdminForgottenPasswordPage.tsx (hunk 1 above)
Comment (MartinEden) [reply app/src/main/admin/components/AdminForgottenPasswordPage.tsx]: Feel like this ticket has enough PRs. So here's a new ticket for either of us to work on: https://vimc.myjetbrains.com/youtrack/issue/VIMC-2020

**Pull request merged**